lib/matplotlib/backend_tools.py

```diff
@@ -589,6 +595,43 @@ class SaveFigureBase(ToolBase):
     default_keymap = rcParams['keymap.save']
 
 
+class ToolSaveFigure(ToolBase):
```

Comment:
I agree, I added it to the Todo list at the top of the PR under TkAgg.  As the code stands at the moment, TkAgg still works in this PR for ``rcParams['toolbar'] == 'toolmanager'``.